### DISCUSSION TITLE
Can I get script of videos? I want to translate to Korean!

### DISCUSSION THREAD

**Discussion by petanerd** [OpenMined/courses#82] [Feedback] [answered]
Hi my name is Dongjun Min from South Korea.

Thanks for make this awesome lectures.
I love to use Pysyft and big interested in Private AI.

So I highly recommended my friends, colleagues and Korean community to enroll in this course.

However, since the lectures are in English and there is no script, I thought it would be a little difficult for my friends who are not proficient in English.

So, if you provide the script, I would like to translate it into Korean and share it. This is a kind of voluntary and attempting to participate as part of a community.

Oh and finally...
Best wishes for a Happy New Year!

**Comment by mcleonard** [comment, marked as answer]
Hi there! We're on this 👍

**Reply by mcleonard** [reply]
Tracking here: https://github.com/OpenMined/courses/issues/99
